src/Slices/ServiceInstanceDetails/UI/Components/CompileReportLink.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from "react";
+import { Link } from "react-router-dom";
+import { Button } from "@patternfly/react-core";
+import { DependencyContext } from "@/UI/Dependency";
+import { words } from "@/UI/words";
+
+interface Props {
+  compileId: string;
+  isExport: boolean;
+}
+
+/**
+ * CompileReportLink Component
+ *
+ * A plain link button that displays the text "Export" if it's an export compile link or "Validation".
+ * The difference between the two compiles can be spotted in the message present in the details of the event.
+ * A validation compile should always contain a key word validate/validation.
+ *
+ * @prop {Props} props - The props of the CompileReportLink
+ *  @prop {string} compileId - The id of the compile required to construct the link towards the Compile details page.
```

Comment:
```suggestion
 * @prop {string} compileId - The id of the compile required to construct the link towards the Compile details page.
```

Reply:
The indent is on purpose, I've been doing it everywhere honestly out of habit to have some sort of hierarchy, but I can adjust it if needed

Reply:
Oh okay, no need then I just thought It was off :D I don't mind having it or not, I personally keep them alligned :D